web/src/beta/features/PluginPlayground/Plugins/presets/index.ts

```
@@ -24,6 +24,12 @@ import { filterFeatureWithStyle } from "./manageLayerStyle/filterFeaturebyStyle"
 import { layerStylingExamples } from "./layerStyles/layerStylingExamples";
 import { overrideStyle } from "./manageLayerStyle/overrideStyle";
 import { styleWithCondition } from "./manageLayerStyle/styleWithCondition";
+import { zoomInOut } from "./camera/zoomInOut";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'zoomInOut'. Consider to delete it or rename it.

'zoomInOut' is defined here:


(lint/suspicious/noRedeclare)